framework/tests/src/tests/AnnotatedForTest.java

```diff
@@ -17,7 +17,7 @@ public AnnotatedForTest(File testFile) {
               "subtyping",
               "-Anomsgtext",
               "-Aquals=tests.util.SubQual,tests.util.SuperQual",
-              "-AuseSafeDefaultsForUnannotatedSourceCode");
+              "-AuseDefaultsForUncheckedCode=source,bytecode");
```

Comment:
It doesn't need to be turned off. What I meant was that the only test that currently exists in the conservative-defaults/annotatedfor directory is testing unchecked code defaults for source code only, and it is self-contained (i.e. it does not make use of any libraries with unchecked bytecode). That said, on further thought my suggestion was too aggressive. Just writing "-AuseDefaultsForUncheckedCode=source" would convey that only unchecked code defaults for source code are being tested, and that the setting for bytecode is not relevant.